lint.js

```diff
@@ -2,19 +2,23 @@
 
 const fs = require("fs").promises;
 
+const isString = obj =>
```

Comment:
`typeof spec === "string"` seems like it could replace this too.

Reply:
Done.